frontend/src/schemas/SchemaList.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, List, ListItem, ListItemText } from '@mui/material'
+import { useGetSchemas } from 'actions/schema'
+import Loading from 'src/common/Loading'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+
+interface SchemaDisplayProps {
+  schemaKind: string
+}
+
+export default function SchemaList({ schemaKind }: SchemaDisplayProps) {
+  const { schemas, isSchemasLoading, isSchemasError } = useGetSchemas(schemaKind)
+
+  const error = MultipleErrorWrapper('Unable to load schemas', {
+    isSchemasError,
+  })
+  if (error) return error
+
+  return (
+    <>
+      {isSchemasLoading && <Loading />}
+      <List>
+        {schemas.map((schema, index) => (
+          <ListItem
+            key={schema.id}
+            divider={index + 1 !== schemas.length}
+            disableGutters
+            secondaryAction={
+              <>
+                <Button disabled>Edit</Button>
+                <Button disabled>{schema.active ? 'Set as inactive' : 'Mark as active'}</Button>
+              </>
+            }
+          >
+            <ListItemText
+              primary={schema.name}
+              primaryTypographyProps={{}}
+              sx={{ maxWidth: 'fit-content', wordBreak: 'break-all', pr: 20 }}
```

Comment:
This is needed so that long text doesn't overlap the buttons on smaller screens:

![image](https://github.com/gchq/Bailo/assets/89992537/8517b4bd-3ebc-4730-a6e2-a1d8737bfa29)